downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/MolecularFunctionAnnotationBuilder.java

```diff
@@ -127,7 +129,7 @@ private static void processProteins(Set<GKInstance> proteinInstances, GKInstance
                 String taxonIdentifier = ((GKInstance) speciesInst.getAttributeValue(ReactomeJavaConstants.crossReference)).getAttributeValue(ReactomeJavaConstants.identifier).toString();
                 if (!GOAGeneratorUtilities.isExcludedMicrobialSpecies(taxonIdentifier)) {
                     if (catalystInst.getAttributeValue(ReactomeJavaConstants.activity) != null) {
-                        getGOMolecularFunctionLine(catalystInst, referenceEntityInst, taxonIdentifier, reactionInst);
+                        goaLines.add(generateGOMolecularFunctionLine(catalystInst, referenceEntityInst, taxonIdentifier, reactionInst));
```

Comment:
Similar to CellularComponent where a `validateProtein` method could help emphasize the line generation/addition over the logging.